Add necessary member in CTransaction #130

lbyyy opened the pull request:
For confidential tx.

EncryptKeys : key to decript the data.

hihiben commented on src/primitives/transaction.h:
```diff
@@ -216,10 +216,12 @@ class CTransaction
     const std::vector<CTxOut> vout;
     const uint32_t nLockTime;
     const tx_type type;
+    const std::vector<std::string> EncryptKeys;
```
variable第一個字小寫，然後這個叫encryptedKeys會不會好一些

hihiben commented on src/primitives/transaction.h:
```diff
@@ -216,10 +216,12 @@ class CTransaction
     // and bypass the constness. This is safe, as they update the entire
     // structure, including the hash.
     const int32_t nVersion;
+    const bool fEncrypted;
```
直接拿EncryptKeys的vector size來判斷是否為encrypted就可以了吧

lbyyy committed:
Add a member in CTransaction #130

EncryptKeys : key to decript the data.
https://github.com/OpenNetworking/gcoin-community/issues/130
